src/TransferableStorage.php

```diff
@@ -22,13 +22,21 @@ interface TransferableStorage
 {
     /**
      * Get messages from the storage into the $catalogue.
+     *
+     * @var array a list of arbitrary options that could be used. The array SHOULD
+     *            use a format of array<string, array<mixed $value>.
+     *            Example: ['foo' => ['bar', 'baz]]
      */
     public function export(MessageCatalogueInterface $catalogue, array $options = []): void;
 
     /**
      * Populate the storage with all the messages in $catalogue. This action
      * should be considered as a "force merge". Existing messages in the storage
      * will be overwritten but no message will be removed.
+     *
+     * @var array a list of arbitrary options that could be used. The array SHOULD
+     *            use a format of array<string, array<mixed $value>.
```

Comment:
Isn't value should be `string` as well instead of `mixed`? Anyway, the given format is confusing for me :/ though the example helps a lot. Probably better format is:
`array[string $key => array[mixed $value]]`?

Wdyt guys? Or is there an official format description somewhere that suggests exactly your syntax?

Reply:
Thank you. 

> Isn't value should be string as well instead of mixed? 

Not really, the `$value` could be an array or an object. 

> Or is there an official format description

I think this is the official syntax. Ie, the one phpstan and other parsers are using. But I dont find any references to it..

Reply:
Ah, ok, thanks for the explanation! looks good then 👍